language-plutus-core/test/Evaluation/Machines/Budget/Examples/InterList/FoldrInterList.plc.golden

```diff
@@ -1,4 +1,4 @@
-((Left An error has occurred:  User error:
+( (Left An error has occurred:  User error:
```

Comment:
The `PrettyBy config (a, b)` instance previously defined in `PlutusPrelude` wasn't behaving the same way the `Pretty (a, b)` instance behaves. I've fixed that, hence the changes.